src/main/webapp/app/exercises/programming/assess/manual-result/programming-assessment-manual-result.service.ts

```diff
@@ -20,11 +20,14 @@ export class ProgrammingAssessmentManualResultService {
      * @param {submit} submit - Indicates whether submit or save is called
      */
     // TODO: make consistent with other *.assessment.service.ts file
-    saveAssessment(participationId: number, result: Result, submit = false): Observable<EntityResponseType> {
+    saveAssessment(participationId: number, result: Result, submit = false, correctionRound = 0): Observable<EntityResponseType> {
         let params = new HttpParams();
         if (submit) {
             params = params.set('submit', 'true');
         }
+        if (correctionRound !== 0) {
```

Comment:
Maybe this condition is stronger? correctionRound > 0 since here it's possible to input negative numbers